userspace/libsinsp/container.cpp

```diff
@@ -83,6 +83,58 @@ const sinsp_container_info::container_mount_info *sinsp_container_info::mount_by
 	return NULL;
 }
 
+std::string sinsp_container_info::normalize_healthcheck_arg(const std::string &arg)
+{
+	std::string ret = arg;
+
+	if(ret.empty())
+	{
+		return ret;
+	}
+
+	// Remove pairs of leading/trailing " or ' chars, if present
+	while(ret.front() == '"' || ret.front() == '\'')
+	{
+		if(ret.back() == ret.front())
+		{
+			ret.pop_back();
+			ret.erase(0, 1);
+		}
+	}
+
+	return ret;
+}
+
+void sinsp_container_info::parse_healthcheck(const Json::Value &healthcheck_obj)
+{
+	if(!healthcheck_obj.isNull())
+	{
+		const Json::Value &test_obj = healthcheck_obj["Test"];
+
+		if(!test_obj.isNull() && test_obj.isArray() && test_obj.size() >= 2)
+		{
+			if(test_obj[0].asString() == "CMD")
+			{
+				m_has_healthcheck = true;
```

Comment:
```m_has_healthcheck = true;``` is common, it may be moved before or after the if/elseif

Reply:
Well there's a possibility that the first item in the array could be something other than CMD/CMD-SHELL, in which case we should do nothing. So I think it makes sense to keep setting this bool inside each case.